### PR TITLE
User-can-delete-an-entry-Bishan-Webeshete

### PR DESCRIPTION
**Pull request by BishanWebeshete**
Kapture 2023-02-18 at 14 13 34

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -127,6 +178,10 @@ i {
   #entry-form-tag {
     margin-left: 22rem;
   }
+
+  .save-container {
+    padding-left: 35.5rem;
```
Comment (shawnkost) [inline css/styles.css]: You should avoid using such large rem values, you should really only be going up to about `5rem`. If you have such large values, then you're probably doing something "hacky" to get this to work

File: js/main.js [hunk 2]
```diff
@@ -137,6 +143,28 @@ function viewSwap(view) {
...
+  $modalContainer.classList.add('hidden');
+});
+
+$confirmButton.addEventListener('click', function (event) {
+  $modalContainer.classList.add('hidden');
+  viewSwap('entries');
+  for (let i = 0; i < data.entries.length; i++) {
+    if (data.editing === data.entries[i]) {
+      data.entries.splice(i, 1);
+      data.nextEntryId--;
```
Comment (shawnkost) [inline js/main.js]: You shouldn't be decreasing `nextEntryId`, you only want this value to stay the same, or to increase

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes requested here Bishan!

**Comment by BishanWebeshete** [comment]
Kapture 2023-02-20 at 19 02 25

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**